packages/core/src/services/experiments/assertRequirements.ts

```diff
@@ -9,12 +8,12 @@ export function assertEvaluationRequirements({
 }: {
   evaluations: EvaluationV2[]
   datasetLabels: Record<string, string>
-}): TypedResult<undefined, LatitudeError> {
+}) {
   for (const evaluation of evaluations) {
     const spec = getEvaluationMetricSpecification(evaluation)
     if (!spec.supportsBatchEvaluation) {
       return Result.error(
-        new BadRequestError(
+        new Error(
```

Comment:
LatitudeError 🙏🏼

Reply:
This is used inside a transaction. And transactions only understand `Error`